fluentd/run.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+OCP_OPERATIONS_PROJECTS=${OCP_OPERATIONS_PROJECTS:-"default openshift openshift-"}
+OCP_FLUENTD_TAGS=('journal.system.** system.var.log** audit.log**')
+for p in ${OCP_OPERATIONS_PROJECTS}; do
+    OCP_FLUENTD_TAGS+=("_**${p}_**")
```

Comment:
should be
```
 +    OCP_FLUENTD_TAGS+=("**_${p}_**") 
```
and - one of the tags will be `**_openshift-_**` which will match nothing - I think you need it to be `**_openshift-*_**`

This is going to be tricky . . .